Bottom: 04ade1c429d0971a1ce7e76121a4a911b8849a81
Top:    881ad0c3480b284844d445d1dffce2580157bde6
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-22 14:46:15 -0500

Fix mariadb bind ip address


---
diff --git a/roles/mariadb/tasks/main.yml b/roles/mariadb/tasks/main.yml
index 70e29ec6..b27a0af7 100644
--- a/roles/mariadb/tasks/main.yml
+++ b/roles/mariadb/tasks/main.yml
@@ -42,15 +42,3 @@
     permanent: true
     state: enabled
   when: mariadb_open_firewall
-- name: Set host ssh ListenAddress
-  template:
-    src: mariadb-bind-address.j2
-    dest: /etc/my.cnf.d/mariadb-bind-address.cnf
-  when: ansible_br0 is defined and ansible_br0.ipv4 is defined and ansible_br0.ipv4.address is defined
-
-- name: Restart MariaDB service
-  service:
-    name: mariadb
-    enabled: yes
-    state: restarted
-  when: ansible_br0 is defined and ansible_br0.ipv4 is defined and ansible_br0.ipv4.address is defined
diff --git a/roles/mariadb/templates/mariadb-bind-address.j2 b/roles/mariadb/templates/mariadb-bind-address.j2
index f30ea73a..831edde0 100644
--- a/roles/mariadb/templates/mariadb-bind-address.j2
+++ b/roles/mariadb/templates/mariadb-bind-address.j2
@@ -1,3 +1,3 @@
 # Bind the database only to the host port
 [mysqld]
-bind-address = {{ ansible_br0["ipv4"]["address"] }}
+bind-address = {{ ansible_default_ipv4["address"] }}
